themes/custom/droopler_theme/scss/components/_d_p.scss

```diff
@@ -17,4 +21,16 @@ $d-p-long-text-margin-bottom: 1.5rem !default;
       margin-bottom: 0;
     }
   }
+
+  // Typically, we don't want any margin on the last d_long_text field.
+  // However, there are few exceptions that need separate handling.
+  // Add "paragraph--with-long-text-margin" class to your paragraph
```

Comment:
@grzegorz-pietrzak-droptica shouldn't it be `Add "with-long-text-margin" class to your paragraph`

Reply:
Ahh, I changed it later :-) Will fix

Reply:
ups, btw "with-forced-long-text-margin"
:yoda: